include/cuco/detail/static_map/static_map.inl

```diff
@@ -54,6 +54,35 @@ constexpr static_map<Key, T, Extent, Scope, KeyEqual, ProbingScheme, Allocator,
 {
 }
 
+template <class Key,
+          class T,
+          class Extent,
+          cuda::thread_scope Scope,
+          class KeyEqual,
+          class ProbingScheme,
+          class Allocator,
+          class Storage>
+constexpr static_map<Key, T, Extent, Scope, KeyEqual, ProbingScheme, Allocator, Storage>::
+  static_map(Extent n,
```

Comment:
Instead of having this second ctor implementation, can the ctor taking an `n`+`load_factor` just compute the capacity and then call the ctor that takes a `capacity`?

Reply:
Probably not, this overload will convert all input extent into dynamic extent (since load factor is not a build time constant) thus calling the `capacity` overload won't work out.

Reply:
Damn, I haven't thought about this. Good point then.